backend/lib/endpoints/schema/users.js

```diff
@@ -5,6 +5,19 @@ const getUserByIdSchema = {
   params: strictSchema().prop("userId", S.string().required()),
 };
 
+const createProfileSchema = {
+  body: S.object()
+    .additionalProperties(false)
+    .prop("firstName", S.string().required())
+    .prop("lastName", S.string().required())
+    .prop("email", S.string().required())
```

Comment:
I think we should not allow to pass this `email` prop in the request for now.
The email can be used to create an account with Auth0, and then it will verify the email through their service.
For social media profiles, most of them will have confirmed email addresses, except Twitter and maybe some rare Facebook profiles... but I think we need to find an alternative solution for that, to make sure we only have verified email addresses in our database.
I'm even wondering if it's a good idea to have the email in our own database, because it's already stored in Auth0, maybe we could just encode it in the JWT as well, what do you think about this @Naraujo13 ?
For now we could also do `Auth0.getUser(token);` like in the `/auth/login` and then use the email that we get back from Auth0 and store it in our database.

Reply:
I agree that there is no need for `email` prop in the request, getting it through Auth0 seems best. 
About storing or not on our side, I see no issue in storing it

Reply:
@Naraujo13 the reasons that we might prefer not to store it in our db:
- avoid duplication (also need to sync when changing etc, potential bugs)
- more secure to let auth0 handle user data... if our db gets hacked they don’t have the emails